ecf/scripts/prep/cam/jevs_cam_hrrr_severe_prep_vhr_master.ecf

```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:10:00
```

Comment:
Job ran for ~1 min in my testing and I see in the parallel the max runtime it has is also ~1 min. Can we make the walltime 10 min to follow NCO's guideline for jobs run 0-2 min to have a walltime of 10 min?

Reply:
Looks like it already is 10 minutes.  I edited other severe jobs to match their ecf scripts.  Maybe we can run those too?
- jevs_cam_hireswarw_severe_prep
- jevs_cam_hireswarwmem2_severe_prep
- jevs_cam_hireswfv3_severe_prep
- jevs_cam_namnest_severe_prep

Reply:
Sure! Should I do `qsub -v vhr=12` to submit them?

Reply:
Yes, thanks!